game.go

```diff
@@ -115,6 +115,10 @@ type Gamer interface {
 	initGame(sprites []Spriter) *Game
 }
 
+var (
+	gameInstance *Game
```

Comment:
Is it possible to implement with `Gopt` to avoid the singleton?

I think it is better for spx to not assume only one game instance exists at the same time.

Reply:
ok